pkg/gui/keybindings/keybindings.go

```diff
@@ -170,6 +174,10 @@ func LabelFromKey(key types.Key) string {
 func GetKey(key string) types.Key {
 	runeCount := utf8.RuneCountInString(key)
 	if runeCount > 1 {
+		// indicates that action has no operations
+		if strings.ToLower(key) == nop {
+			return []rune(key)[0]
```

Comment:
I don't understand this line. It looks like it's saying that if our key is '<nop>' then we return '<' but I don't know why we would do that. I would instead return nil